Sources/FluentUI_common/Core/Theme/FluentTheme+Tokens.swift

```diff
@@ -120,6 +120,12 @@ public extension FluentTheme {
         case presenceDnd
         case presenceAvailable
         case presenceOof
+
+        // MacOS Only Tokens
+        @available(iOS, unavailable)
+        @available(visionOS, unavailable)
+        @available(macCatalyst, unavailable)
+        case background4Hover, foreground4
```

Comment:
Nit: could you leave this in place with the other `background4` tokens? As we add more tokens that are platform-specific, I think it'll be easier to keep them organized by category.